File: python/goa_legislative.py
# Author : Mustafa Mulla
# Roll No : 27
from abc import ABC, abstractmethod
from typing import List

class GoaLegislative(ABC):
    def __init__(self, section_name: str):
        self._section_name = section_name

    def get_section_name(self) -> str:
        return self._section_name

    @abstractmethod
    def get_subsections(self) -> List['Subsection']:
        pass

File: java/Goa_legislative.java
// Author : Mustafa Mulla
// Roll No : 27
// Title : Goa legistalive assembly
// Start date : 15/07/2024
// Modified date :  22/07/2024
// Description : Goa Legislative Assembly's various sections
import java.util.List;

public abstract class Goa_legislative {
    private String sectionName;

    public Goa_legislative(String sectionName){
        this.sectionName=sectionName;
    }

    public String getSectionName(){
        return sectionName;
    }

    public abstract List<Subsection> getSubsections();
}


